Misc/NEWS.d/next/Core_and_Builtins/2024-09-19-13-17-31.gh-issue-122878.4iFpsB.rst

```diff
@@ -0,0 +1 @@
+Use the ``pager`` binary on Debian and derivatives, to display REPL ``help()``.
```

Comment:
Are we sure it's Debian-specific?

```suggestion
Use the ``pager`` binary, if available, to display REPL ``help()``.
```

Reply:
Mixture of both sentence:

```
Use the ``pager`` binary, if available (on Debian and derivatives), to display REPL ``help()``.
```

Reply:
Updated, with a tiny change.

Reply:
The Debian alternatives system provides a `pager` binary managed by alternatives, to select the best installed pager. I'm not aware of any other platforms that have a `pager` binary.